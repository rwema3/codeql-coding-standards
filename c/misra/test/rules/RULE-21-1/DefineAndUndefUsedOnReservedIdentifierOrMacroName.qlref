rules/RULE-21-1/DefineAndUndefUsedOnReservedIdentifierOrMacroName.ql