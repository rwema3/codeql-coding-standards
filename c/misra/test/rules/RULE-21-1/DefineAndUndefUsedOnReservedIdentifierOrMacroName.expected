| test.c:1:1:1:17 | #define _NOT_OKAY | Reserved identifier '_NOT_OKAY' has been undefined or redefined. |
| test.c:2:1:2:16 | #undef _NOT_OKAY | Reserved identifier '_NOT_OKAY' has been undefined or redefined. |
| test.c:5:1:5:13 | #define errno | Reserved identifier 'errno' has been undefined or redefined. |
